client/src/App.js

```diff
@@ -3,12 +3,13 @@ import { BrowserRouter, Routes, Route } from "react-router-dom";
 import LoginPage from "./pages/CredentialsPage/LoginPage.js";
 import SignUpPage from "./pages/CredentialsPage/SignUpPage.js";
 import PageNotFound from "./pages/PageNotFound/PageNotFound.js";
+import TaskFormPage from "./pages/TaskFormPage/TaskFormPage.js";
 
 const App = () => {
   return (
     <BrowserRouter>
       <Routes>
-        <Route path="/" element={<LoginPage />} />
+        <Route path="/" element={<TaskFormPage />} />
```

Comment:
There's a conflict with `App.js` when trying to merge with develop, so this will have to be fixed (You can merge the `develop` branch into this one and fix any conflict issues)